README.md

```diff
@@ -208,7 +208,31 @@ At a minimum, all Roaring implementations should be able to parse the two files
 
 ## Unsigned integers in Java
 
-
 Java lacks native unsigned integers, but integers are still considered to be unsigned within Roaring, and ordered  according to  ``Integer.compareUnsigned``. 
 
 For integers in [0,2147483647], the unsigned and signed integers are undistinguisable since Java relies a 32-bit two's complement format for its ``int`` type.
+
+# Extention for 64B implementations
+
+Some Roaring bitmap implementations may offer a 64-bits implementation. This section proposes a portable format, compatible with some (but not all) 
+64-bits implementations. This format is naturally compatible with implementations base on a conventional
```

Comment:
based on ?

Reply:
You refer to `naturally compatible ` ? Because the serialization format follows the in-memory layout (hence why CRoaring and Java chose this format from the beginning).

Reply:
I would write "This format is naturally compatible with implementations *based* on a conventional" and not "This format is naturally compatible with implementations *base* on a conventional".